ggml/src/ggml-vulkan.cpp

```diff
@@ -1743,16 +1743,21 @@ static vk_device ggml_vk_get_device(size_t idx) {
         vk11_features.sType = VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_VULKAN_1_1_FEATURES;
         device_features2.pNext = &vk11_features;
 
+        VkPhysicalDevice16BitStorageFeatures storage_16bit;
+        storage_16bit.pNext = nullptr;
+        storage_16bit.sType = VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_16BIT_STORAGE_FEATURES;
+        vk11_features.pNext = &storage_16bit;
```

Comment:
Doing this is against the [Vulkan Specification](https://registry.khronos.org/vulkan/specs/1.3-extensions/html/vkspec.html#VUID-VkDeviceCreateInfo-pNext-02829) and causes a validation error when run with Validation layers enabled:
```
vkCreateDevice():  If the pNext chain includes a VkPhysicalDeviceVulkan11Features structure, then it must not include a VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_16BIT_STORAGE_FEATURES structure.
```
I'm not sure how to handle this properly, we could avoid `VkPhysicalDeviceVulkan11Features` entirely, but I don't want to do that just cause of Qualcomm driver issues. Do you have an idea?